sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/Statistics.scala

```diff
@@ -155,6 +156,8 @@ object ColumnStat extends Logging {
   private val KEY_NULL_COUNT = "nullCount"
   private val KEY_AVG_LEN = "avgLen"
   private val KEY_MAX_LEN = "maxLen"
+  val KEY_HISTOGRAM = "histogram"
+  val KEY_HISTOGRAM_SEPARATOR = "-"
```

Comment:
private val?

Reply:
they are used in `HiveExternalCatalog` for stats/properties conversion.